dll/win32/browseui/shellfind/CSearchBar.cpp

```diff
@@ -138,27 +137,22 @@ LRESULT CSearchBar::OnSearchButtonClicked(WORD wNotifyCode, WORD wID, HWND hWndC
 
     // See if we have a szFileName, our first character is not an asterisk,
```

Comment:
I think this comment needs some update.
Namely, it should now mention the following points:
- If the szFileName contains explicit wildcards (i.e. the asterisk or the question-mark), a search for this exact search pattern is performed.
- Otherwise if the szFileName do not contain wildcards, a search for all files containing the szFileName in their name is done (i.e. it's equivalent to searching for `*<the_file_name>*` ).

Reply:
Yep. Sorry I missed fixing that. I will take care of it now. Thanks.

Reply:
Updated. Thanks.